packs/ancestries/sprite.json

```
@@ -69,7 +69,7 @@
             "remaster": false,
             "title": "Pathfinder Lost Omens Ancestry Guide"
         },
-        "reach": 0,
+        "reach": 5,
```

```suggestion
        "reach": 0,
```
This keeps happening, might be an issue with the extractor?

---

It is, yeah